src/modules/control_allocator/ActuatorEffectiveness/ActuatorEffectivenessMultirotor.cpp

```diff
@@ -173,6 +175,10 @@ ActuatorEffectivenessMultirotor::computeEffectivenessMatrix(const MultirotorGeom
 		float ct = geometry.rotors[i].thrust_coef;
 		float km = geometry.rotors[i].moment_ratio;
 
+		if (fabsf(ct) < FLT_EPSILON) {
+			continue;
```

Comment:
To make sure the actuator count is correct. The (new) matrix is already 0-initialized, so I don't see a problem here.

Reply:
Yes but it will take a small refactor on the caller side to break it, (for example if the effectiveness matrix is taken by reference from the _control_allocation object in `ControlAllocator::update_effectiveness_matrix_if_needed()`, instead of being created each time). 

Considering this is run on param update only. I do not see a problem zero-ing the matrix column.

Reply:
Fair point, I set the whole matrix to 0 now.